BlueEastCode/bluerain-os
# Feature/ssr mode

Issue:

## What I did

## How to test

Is this testable with jest or storyshots?

Are any of the existing tests are failing ?

Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.


@bjavaid @amnajunaid please review the changes and merge them.

test PR to check notification
